CDS/scripts/js_utils/media-atom-lib.js

```diff
@@ -64,6 +75,17 @@ function fetchMetadata(connection) {
               }, "");
           }
 
+          for(const fieldName of expectedFields){
+              if(! response.hasOwnProperty(fieldName)){
+                  const str = "Missing " + fieldName + " in Media Atom data";
+                  if(process.env.abort_if_missing) {
```

Comment:
don't think we need this flag as we should always quit if required fields are missing